docs/supported-technologies.asciidoc

```diff
@@ -117,6 +117,12 @@ The agent supports <<framework-integrations,integrations with certain frameworks
 To instrument custom metrics, like rendering time or mounting time of components on frameworks like React, Angular, Vue,
 etc., use the <<custom-transactions,custom transactions API>>.
 
+The core RUM agent supports MPAs and SPAs out of the box. This means that for SPAs to use <<framework-integrations,framework specific integrations>> is a recommendation rather than a requirement. We recommend them for the following reasons:
```

Comment:
Might be overkill and obvious, but maybe add some detail for clarity, for example:
> ...supports websites based on multi-page (MPA) and single page (SPA) architectures, out of the box...